src/python/nimbusml/tests/model_summary/test_model_summary.py

```diff
@@ -107,6 +109,18 @@ def test_model_summary_not_supported(self):
             pipeline.fit(train_stream, label_column)
             assert_raises(TypeError, pipeline.summary)
 
+    def test_model_summary_not_supported_specific(self):
+        path = get_dataset('infert').as_filepath()
+        data = FileDataStream.read_csv(path, sep=',',
+                               names={0: 'row_num', 5: 'case'})
+        pipeline = Pipeline([
+            OneHotVectorizer(columns={'edu': 'education'}),
+            FactorizationMachineBinaryClassifier(feature=['induced', 'edu', 'parity'],
+                                         label='case')
+        ])
+        pipeline.fit(data)
+        assert_raises(TypeError, pipeline.summary)
+
```

Comment:
Is this method necessary? It looks like it does the same test as `test_model_summary_not_supported`.

Reply:
I added this specific case of a test on Gani's request earlier to show what the FactorizationMachine error message is now.